knowledge-base/grid-multiple-child-grids-per-item.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Multiple Details Grids per Master
+description: How to render multiple child (detail) Grids under a master Grid?
+type: how-to
+page_title: Show Multiple Detail Grids per Master
+slug: grid-kb-multiple-child-grids-per-item
+position: 
+tags: grid, hierarchy, multiple, children
+ticketid: 1550340
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>Grid for Blazor</td>
+		</tr>
+	</tbody>
+</table>
+
+
+## Description
+
+We have a hierarchical Grid where each item has two sub-Grids. We want to display both child Grids simultaneously.
+
+## Solution
+
+The [hierarchy in Grid]({%slug components/grid/features/hierarchy%}) is achieved through declaring a child Grid in a [`DetailTemplate`](https://docs.telerik.com/blazor-ui/api/Telerik.Blazor.Components.TelerikGrid-1#collapsible-Telerik_Blazor_Components_TelerikGrid_1_DetailTemplate). As the `DetailTemplate` is essentially a `RenderFragment`, you can, in fact, render any desired content there - be that one, two or more Grids, other components or html elements.
```

Comment:
```suggestion
[Grid hierarchy]({%slug components/grid/features/hierarchy%}) requires a child Grid in a [`DetailTemplate`](https://docs.telerik.com/blazor-ui/api/Telerik.Blazor.Components.TelerikGrid-1#collapsible-Telerik_Blazor_Components_TelerikGrid_1_DetailTemplate). The `DetailTemplate` is essentially a `RenderFragment`, so it can include any desired content there - be that one, two or more Grids, other components or html elements.
```